cloud-formation/account-frontend-cf.yaml

```diff
@@ -0,0 +1,227 @@
+      AwsKeyARN: arn:aws:kms:eu-west-1:865473395570:key/89f17663-c79e-4104-b718-ee0986402ad9
+      DomainEnvVariable: code.dev-theguardian.com
+    PROD:
+      MaxInstances: 6
+      MinInstances: 3
+      InstanceType: t2.small
+      CertificateARN: arn:aws:acm:eu-west-1:865473395570:certificate/e4c03b58-bf76-44c2-ae6a-17f81c10eb6f # this is the CODE cert, and will need to be updated before PROD deployment
+      AwsKeyARN: arn:aws:kms:eu-west-1:865473395570:key/d7aed06c-e961-4078-8604-0aeedae08613
+      DomainEnvVariable: theguardian.com
+Resources:
+  AutoScalingGroup:
+    Type: AWS::AutoScaling::AutoScalingGroup
+    Properties:
+      AvailabilityZones: !GetAZs
+      VPCZoneIdentifier: !Ref Subnets
+      LaunchConfigurationName: !Ref LaunchConfig
+      MinSize: !FindInMap [ StageVariables, !Ref Stage, MinInstances ]
+      MaxSize: !FindInMap [ StageVariables, !Ref Stage, MaxInstances ]
+      TargetGroupARNs:
+        - !Ref TargetGroup
+      HealthCheckType: ELB
+      HealthCheckGracePeriod: 300
+      Tags:
+        - Key: Stack
+          Value: !Ref Stack
+          PropagateAtLaunch: true
+        - Key: App
+          Value: !Ref App
+          PropagateAtLaunch: true
+        - Key: Stage
+          Value: !Ref Stage
+          PropagateAtLaunch: true
+  LaunchConfig:
+    Type: AWS::AutoScaling::LaunchConfiguration
+    Properties:
+      ImageId: !Ref AMI
+      SecurityGroups:
+        - !Ref InstanceSecurityGroup
+      InstanceType: !FindInMap [ StageVariables, !Ref Stage, InstanceType ]
+      IamInstanceProfile: !Ref InstanceProfile
+      AssociatePublicIpAddress: true
+      UserData:
+        Fn::Base64: !Sub
+          - |+
+            #!/bin/bash -ev
+
+            # get and install node
+            wget -nv https://nodejs.org/dist/v9.11.1/node-v9.11.1-linux-x64.tar.xz -P /tmp
+            tar -xf /tmp/node-v9.11.1-linux-x64.tar.xz --directory /usr/local/
+
+            # get runnable tar from S3
+            aws --region ${AWS::Region} s3 cp s3://membership-dist/${Stack}/${Stage}/${App}/account-frontend.tgz /tmp
+            mkdir /etc/gu
+            tar -xvzf /tmp/account-frontend.tgz --directory /etc/gu/
+
+            # set DOMAIN environment variable based on StageVariables
+            export DOMAIN=${DomainEnvVariable}
+
+            # RUN
+            /usr/local/node-v9.11.1-linux-x64/bin/node /etc/gu/dist/server.js
+
+          - DomainEnvVariable: !FindInMap [ StageVariables, !Ref Stage, DomainEnvVariable ]
+
+  AppRole:
+    Type: AWS::IAM::Role
+    Properties:
+      Path: "/"
+      ManagedPolicyArns: [ "arn:aws:iam::aws:policy/service-role/AmazonEC2RoleforSSM" ]
+      AssumeRolePolicyDocument:
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - ec2.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Policies:
```

Comment:
We should not need the permissions to use fetch these keys from S3 (due to GDPR, we are changing the way that we control ssh access to instances).